lib/gherkin_lint.rb

```diff
@@ -31,69 +33,52 @@
 require 'gherkin_lint/linter/use_outline'
 require 'multi_json'
 require 'set'
-
+require 'gherkin_lint/configuration'
```

Comment:
could there be both usages possible? the cli way and the config way? how to express a pattern for your require_tags switch?